packages/db/schema.prisma

```diff
@@ -12,6 +12,12 @@ datasource db {
   relationMode = "prisma"
 }
 
+enum NotificationType {
```

Comment:
Can we add a miscellaneous type jic we add some more notifications in the future for something like you earned a badge or any news we want to share through notifications and so on.

enum NotificationType {
  LIKE
  REPLY
  MENTION
  MISC
}

Reply:
Some other options:

- `SYSTEM` used for critical updates to users
- `NEW_FOLLOWER` if we wanna be social 

and I agree with @zaCKoZAck0 about a `MISC` as a catch all.

Reply:
i like SYSTEM. we can add follows at some point but doubt that'll be a thing for a bit